### PR TITLE
fix(apu): Added xfeed APU capabilities

### PR DESCRIPTION
**Pull request by Taz5150**
Fixes #[issue_no]

## Summary of Changes

This PR fixes/ adds cross feed APU fuel capabilities.

With xfeed OFF, APU will always burn fuel from the left tank.
With xfeed ON, APU burns fuel from both tanks as per the table below

## Screenshots (if necessary)

image

## Additional context

Discord username (if different from GitHub): TazX [Z+1]#0405

## Testing instructions

Starting with the aircraft C&D:

1. Proceed to start the APU
2. Press Xfeed PB to ON
3. Check fuel is burnt from both Left & Right tanks
4. Turn both Left fuel pumps OFF; check fuel is burnt from Right tank
5. Turn both Left pumps ON
6. Turn both Right pumps OFF; check fuel is burnt from Left tank
7. Turn all fuel pumps OFF; check fuel is burnt from both Left & Right tanks

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: fbw-a32nx/src/wasm/fadec_a320/src/EngineControl.h [hunk 1]
```diff
@@ -1292,4 +1307,4 @@ class EngineControl {
   }
 };
 
-EngineControl EngineControlInstance;
+EngineControl EngineControlInstance;
```
Comment (tracernz) [inline fbw-a32nx/src/wasm/fadec_a320/src/EngineControl.h:1310]: Why make the text file invalid by removing the last line feed? :P

**Review by tracernz — APPROVED** [review]

**Comment by donstim** [comment]
LGTM

**Comment by 2hwk** [comment]
Needs modification to account for fuel burn when left fuel pumps are off (see discussion in #development)

**Comment by Taz5150** [comment]
> Needs modification to account for fuel burn when left and right pumps are off (see discussion in #development)

I have updated the code as per the following table. Still waiting for komp's feedback as per discussion in #development:

image

**Comment by BlueberryKing** [comment]
Dev Team Report

Discord : BlueberryKing
Object of testing: 8216
Tier of Testing : 1
Date : 14/10/2023

Testing Process:
I checked some fuel flow values by making sure the values on the SD page went down, but later just looked at the simvar values because the process was very slow.

- Start C&D
- Proceed to start the APU
- Press Xfeed PB to ON
- Checked fuel is burnt from both Left & Right tanks
- Turned both Left fuel pumps OFF; checked that fuel is burnt from Right tank
- Turned both Left pumps ON
- Turned both Right pumps OFF; checked that fuel is burnt from Left tank
- Turned all fuel pumps OFF; checked fuel is burnt from both Left & Right tanks

Testing Results:
Passed

Media:
N/A

Conclusion:
LGTM!

**Pull request merged**